src/ray/raylet/node_manager.h

```diff
@@ -742,7 +756,9 @@ class NodeManager : public rpc::NodeManagerServiceHandler {
   /// A mapping from actor ID to registration information about that actor
   /// (including which node manager owns it).
   std::unordered_map<ActorID, ActorRegistration> actor_registry_;
-
+  /// A mapping from ObjectIDs to external object URLs for spilled objects.
+  /// TODO(suquark): Move it into object directory.
+  std::unordered_map<ObjectID, std::string> spilled_objects_;
```

Comment:
absl::flat_hash_map

Reply:
are we going to use absl for all of these? I noticed all other members are still using std containers in node manager.

Reply:
Absl maps are considerably faster across the board, we should always prefer to use them whenever possible.